Dockerfile

```
@@ -0,0 +1,36 @@
+ARG DOCKER_BUILDER_SERVER_IMAGE=golang:1.14
+ARG DOCKER_BASE_IMAGE=alpine:3.12
+
+# Build Server
+FROM ${DOCKER_BUILDER_SERVER_IMAGE} AS builder-server
+ARG GITHUB_USERNAME
+ARG GITHUB_TOKEN
+
+WORKDIR /server
+
+# Download dependencies
+RUN echo "machine github.com login ${GITHUB_USERNAME} password ${GITHUB_TOKEN}" > ~/.netrc
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+
+COPY . .
+
```

```suggestion

COPY . .

RUN make build

```
I think you might have forget to run `make build` here. Otherwise building image does not work if binary was not build locally, which makes downloading modules kind of pointless.